recommenders/models/newsrec/models/base_model.py

```diff
@@ -186,6 +186,8 @@ def fit(
         valid_behaviors_file,
         test_news_file=None,
         test_behaviors_file=None,
+        steps=None,
```

Comment:
I think it would be clearer if you call it `step_limit`, otherwise your step will be confused with the step defined in the loop. Also, please add a comment in the docstring defining the step limit

Reply:
okay, sure. I will add it right now